bench/wtperf/runners/evict-btree-readonly.wtperf

```diff
@@ -0,0 +1,12 @@
+# wtperf options file: evict btree configuration
```

Comment:
Is this a performance test, or more of a functional test? If it's a performance test is it going to test something different to the non-read-only evict btree (now or in the future)?